### PR TITLE
Fix/update font weight 700

### PR DESCRIPTION
**Pull request by glespinosa**
New Folder
image

Rename Model
image

Delete Model
image

Delete Folder
image

Resume Session: Update Welcome Text Font Weight to 700 #2422 
image

Content: Unsaved Changes Modal - Update Modal Title Font Weight #2467 
image

### CODE REVIEW

File: src/apps/media/src/app/components/RenameFolderDialog.tsx [hunk 1]
```diff
@@ -57,7 +57,7 @@ export const RenameFolderDialog = ({
 
   return (
     <Dialog open={open} onClose={onClose} fullWidth maxWidth={"xs"}>
-      <DialogTitle>Rename Folder</DialogTitle>
+      <DialogTitle fontWeight={700}>Rename Folder</DialogTitle>
```
Comment (shrunyan) [inline src/apps/media/src/app/components/RenameFolderDialog.tsx:60]: Why is `fontWeight` added to the `DialogTitle` component in this scenario but in the `NewFolderDialog` component an additional `Typography` component was included?

Comment (glespinosa) [reply src/apps/media/src/app/components/RenameFolderDialog.tsx]: My mistake on that one, updating it now

**Comment by theofficialnar** [comment]
I am actually thinking if is this worth making a  theme-level change since I've noticed that @zcolah's dialog designs lately have fontWeight 700 on the titles.

**Comment by glespinosa** [comment]
> I am actually thinking if is this worth making a theme-level change since I've noticed that @zcolah's dialog designs lately have fontWeight 700 on the titles.

Created PR for this in material repo : https://github.com/zesty-io/material/pull/88

File: src/apps/media/src/app/components/RenameFolderDialog.tsx (hunk 1 above)
Comment (glespinosa) [reply src/apps/media/src/app/components/RenameFolderDialog.tsx]: updated the code now to use dialog title instead and made a PR in material repo for updating the font weight on theme level. Thank you
https://github.com/zesty-io/material/pull/88

**Comment by zcolah** [comment]
Please confirm if we update all the dialog titles to be 700 will it overwrite the dialogs (e.g. Save Item) where the font weights are mixed? If yes, I can make a list of the dialogs where the one-off has to be present.

**Comment by zcolah** [comment]
Next Steps 
1. Update the Design System to have the Dialog Title default font weight be 700. 
2. Here is a list of one off modals that have a mix of 600 and 700 that will need to be updated. See below. 

CleanShot 2024-02-22 at 23 33 49

CleanShot 2024-02-22 at 23 33 42

**Comment by glespinosa** [comment]
> Next Steps
> 
> 1. Update the Design System to have the Dialog Title default font weight be 700.
> 2. Here is a list of one off modals that have a mix of 600 and 700 that will need to be updated. See below.
> 
> CleanShot 2024-02-22 at 23 33 49
> 
> CleanShot 2024-02-22 at 23 33 42

Output
1. this will be solve by merging this PR https://github.com/zesty-io/material/pull/88

2. Updating the weight in the manager ui with mix 700 and 600 weights
image
image

**Review by theofficialnar — CHANGES_REQUESTED** [review]
Isn't it unnecessary to put in fontWeight 700 to these modal title considering you already have this pr: https://github.com/zesty-io/material/pull/88 open which will eventually make all the dialog title's have fontWeight of 700 anyway?

**Comment by glespinosa** [comment]
> Isn't it unnecessary to put in fontWeight 700 to these modal title considering you already have this pr: [zesty-io/material#88](https://github.com/zesty-io/material/pull/88) open which will eventually make all the dialog titles have fontWeight of 700 anyway?

The issue is that the text inside of the dialog title is inside of typography so even I make changes in the material repo it won't reflect the changes. unless I remove the text inside of a typography component and copy how the `NewFolderDialog` render texts... And also there's some difference how the <DialogTitle/> is used in `DeleteItemDialog` and `NewFolderDialog`...

**Review by theofficialnar — APPROVED** [review]

File: src/apps/content-editor/src/app/components/PendingEditsModal/PendingEditsModal.tsx [hunk 2]
```diff
@@ -93,7 +93,7 @@ export default memo(function PendingEditsModal(props: PendingEditsModalProps) {
             >
               <WarningAmberRounded color="warning" />
             </Box>
-            <Typography variant="h5" sx={{ mt: 1.5 }}>
+            <Typography variant="h5" fontWeight={700} sx={{ mt: 1.5 }}>
```
[the remark this answers is not in the document]
Comment (glespinosa) [reply src/apps/content-editor/src/app/components/PendingEditsModal/PendingEditsModal.tsx]: updated to use fontWeight='inherit'

File: src/apps/schema/src/app/components/DeleteModelDialogue.tsx [hunk 3]
```diff
@@ -88,7 +88,7 @@ export const DeleteModelDialogue = ({ onClose, model }: Props) => {
         >
           <DeleteRoundedIcon color="error" />
         </Box>
-        <Typography variant="h5" sx={{ mt: 1.5 }}>
+        <Typography variant="h5" fontWeight={700} sx={{ mt: 1.5 }}>
```
[the remark this answers is not in the document]
Comment (glespinosa) [reply src/apps/schema/src/app/components/DeleteModelDialogue.tsx]: updated to use fontWeight='inherit'

File: src/apps/schema/src/app/components/RenameModelDialogue.tsx [hunk 4]
```diff
@@ -77,7 +77,7 @@ export const RenameModelDialogue = ({ onClose, model }: Props) => {
         >
           <DriveFileRenameOutlineRoundedIcon color="info" />
         </Box>
-        <Typography variant="h5" sx={{ mt: 1.5 }}>
+        <Typography variant="h5" fontWeight={700} sx={{ mt: 1.5 }}>
```
[the remark this answers is not in the document]
Comment (glespinosa) [reply src/apps/schema/src/app/components/RenameModelDialogue.tsx]: updated to use fontWeight='inherit'

**Comment by shrunyan** [comment]
@glespinosa & @agalin920 with the change to material. https://github.com/zesty-io/material/pull/88 Should this PR#2551 get updated? As I imagine we don't need to explicitly define 700 font weight any many places and "inherit", as that should be the default behavior.

**Pull request merged**